x-pack/plugins/ml/public/application/components/feedback_button/feature_feedback_button.tsx

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
This is an exact copy of the [observability button component](https://github.com/elastic/kibana/blob/4bad5ce8cb76a266499ba284618d747aa23de990/x-pack/plugins/observability_shared/public/components/feature_feedback_button/feature_feedback_button.tsx).
For ML to use this component it would be best moved to a package.
I believe the creation of this package is outside of the scope of this PR, but as soon as the package is created, we can remove this file.